traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -306,71 +306,71 @@
           },
           "multiSiteOrigin": {
             "title": "Use Multi Site Origin Feature",
-            "desc": "Enable the Multi Site Origin feature for this delivery service.<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#rl-multi-site-origin\" target=\"_blank\">See Multi Site Origin...</a>"
+            "desc": "Enable the Multi Site Origin feature for this delivery service. Is the Multi Site Origin feature enabled for this delivery service (True/False).<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#rl-multi-site-origin\" target=\"_blank\">See Multi Site Origin.</a>"
           },
           "orgServerFqdn": {
             "title": "Origin Server Base URL",
-            "desc": "The Origin Server’s base URL. This includes the protocol (http or https). Example: http://movies.origin.com"
+            "desc": "The Origin Server’s base URL. This includes the protocol (http or https). Example: http://movies.origin.com. Must be a domain only, no directories or IP addresses"
           },
           "originShield": {
             "title": "Origin Shield (Pipe Delimited String)",
-            "desc": "Description required"
+            "desc": "Add another forward proxy upstream of the mid caches. Example: go_direct=true will allow the Mid to hit the origin directly instead of failing if the origin shield is down. Experimental Feature."
           },
           "profileId": {
             "title": "Delivery Service Profile",
-            "desc": "Description required"
+            "desc": "Only used if a delivery service uses configurations that specifically require a profile. Example: MSO configurations or cachekey plugin would require a ds profile to be used."
           },
           "protocol": {
             "title": "Protocol",
-            "desc": "The protocol to serve this delivery service to the clients with.<br><br><a href=\"http://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_ops/using.html#delivery-service\" target=\"_blank\">See more...</a>"
+            "desc": "Services that use \"HTTP and HTTPS\" will be able to deliver both types of traffic."
```

Comment:
This doesn't seem like a good description of "protocol" as it seems to ignore the other types